packages/contractkit/src/identity/offchain/accessors/name.ts

```diff
@@ -0,0 +1,21 @@
+import * as t from 'io-ts'
+import OffchainDataWrapper from '../../offchain-data-wrapper'
+import { EncryptedSimpleSchema, SimpleSchema } from '../schemas'
+
+const NameSchema = t.type({
+  name: t.string,
+})
+
+export type NameType = t.TypeOf<typeof NameSchema>
+
+export class NameAccessor extends SimpleSchema<NameType> {
```

Comment:
It's unclear to me what "Schema" and "Accessor" and defined to mean in this code. Here is seems that an "Accessor" is implements a "Schema", but there is also a "Schema" right above this that describes a data type and does not satisfy the "Schema" interface defined in `utils.ts`.